profiles/base/package.use.stable.mask

```diff
@@ -4,6 +4,12 @@
 # This file requires eapi 5 or later. New entries go on top.
 # Please use the same syntax as in package.use.mask
 
+# Volkmar W. Pogatzki <gentoo@pogatzki.net> (2022-05-08)
```

Comment:
I actually wonder if this mask is really necessary. Ideally we would start a tinder run with system-vm jdk17 and see which packages incorrectly specify a (build) dependency on jdk17.

Reply:
Isn't that mask needed until `virtual/jdk:17` and `virtual/jre:17` are stable on all supported arches?

Reply:
it's nice to have for safer transition, when virtual starts getting stable keywords.